doc/nrf/ug_thingy53.rst

```diff
@@ -0,0 +1,258 @@
+.. _ug_thingy53:
+
+Working with Thingy:53
+######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Nordic Thingy:53 is a battery-operated prototyping platform for IoT Systems.
+
+Thingy:53 integrates the following components:
+
+* nRF5340 SoC - Supporting Bluetooth® Low Energy, 802.14.5 based protocols and Near Field Communication (NFC).
+* nRF21540 RF FEM - Range extender with integrated power amplifier (PA)/low-noise amplifier (LNA).
+* nPM1100 PMIC - Power Management IC with integrated dual-mode buck regulator and battery charger.
+
+For more information on the product, see the `Thingy:53 product page`_ and the `Nordic Thingy:53 User Guide`_.
+The |NCS| provides support for developing applications on the Thingy:53.
+
+.. _thingy53_serialports:
+
+Connecting to Thingy:53 serial ports
+************************************
+
+For connecting to Thingy:53, use a serial terminal.
+
+Thingy:53 uses the following UART baud rate configuration:
```

Comment:
Does that mean the table below this line is not needed?

Reply:
When using USB you do not need to specify the baud rate, so i would remove this table - if we just want to focus on the main way of obtaining logs / having a shell which is using native USB.

Reply:
> When using USB you do not need to specify the baud rate

I think you still do need to specify baudrate on a terminal end. From the tools point of view it is still an UART.

Default for CDC ACM are:
```
115200bps, no parity, 1 stop bit, 8bit char
```

I think we stay on defaults. Right @MarekPieta ?

Reply:
I mean.. I don't put any baudrate which defaults to 9600 on `screen` for example, but any value works (just double-checked). However, if that makes easier life for the user we may leave it.

Reply:
Checked and you are right - it does not matter. On both ends it can be anything.
There is no point in specifying any values here - these will even lead to further confusion. 

Better to note that for CDC ACM baudrate is ignored and transfer goes with USB speed.